client/src/components/Verejne/index.js

```diff
@@ -45,20 +70,25 @@ export default compose(
     (state) => ({
       entities: entitiesSelector(state),
     }),
-    {fetchEntities},
+    {fetchEntities, selectEntity}
   ),
   lifecycle({
     componentDidMount() {
       this.props.fetchEntities()
     },
   }),
   withState('currentPage', 'setCurrentPage', 1),
-  withProps(({entities, currentPage}) => ({
-    entities: chunk(reverse(sortBy(entities, ['size'])), VEREJNE_MAX_PAGE_ITEMS)[currentPage - 1] || [],
-    pageCount: entities ? Math.ceil(entities.length / VEREJNE_MAX_PAGE_ITEMS) : 0,
-  })),
-  branch(
-    (props) => !props.entities.length,
-    renderComponent(Loading),
-  )
+  withProps(({entities, currentPage}) => {
```

Comment:
I see you're already using redux - then please do this with a selector instead of withProps. It both makes more sense organization wise and has the additional benefit of memoization and composability

Reply:
also, I'm kind of missing the flow annotations across the board in this file